document/RuleMetadataProfile.mdk

```diff
@@ -1 +1,165 @@
+that are intended for use in this scenario.
+
+## `runLog` object { #rule-metadata-profile-runLog }
+
+### General { #rule-metadata-profile-runLog-general }
+
+The Rule Metadata Profile defines one additional property in the `runLog` object (&sect;[#runLog]).
+
+### `ruleInfo` property { #ruleInfo }
+
+A `runLog` object (&sect;[#runLog]) may contain a property named `ruleInfo`
+whose value is an array of `ruleDescriptor` (&sect;[#ruleDescriptor]) objects.
+
+## `ruleDescriptor` object { #ruleDescriptor }
+
+### General { #ruleDescriptor-General }
+
+A `ruleDescriptor` object describes a discrete, potentially configurable,
+unit of analysis as implemented by an individual tool.
+
+### `id` property { #ruleDescriptor-id }
+
+A `ruleDescriptor` object shall contain a property named `id` whose value is a string containing
+a stable, opaque identifier for the rule.
+
+~ IsoExample
+`"CA2101"`
+~
+
+### `name` property { #ruleDescriptor-name }
+
+A `ruleDescriptor` object may contain a property named `name` whose value is a string containing
+a readable rule identifier. A rule name is suitable for use as a key to other data or to
+otherwise identify a rule (e.g., within an in-source suppression) but is not guaranteed to
+be stable. In cases where the readable rule `name` contains implementation details that are
+invalidated over time, for example, a tool might alter a rule's readable name (while leaving
+the stable `id` property unchanged).
+
+~ IsoExample
+`"SpecifyMarshalingForPInvokeStringArguments"`
+~
+
+### `abstract` property { #ruleDescriptor-abstract }
+
+A `ruleDescriptor` object may contain a property named `abstract` whose value is a string containing
+a concise description of the rule. The `abstract` property should be a single sentence that is understandable
+when displayed in user interface contexts where the available space is limited to a single line of text.
+
+~ IsoExample
+`"Specify marshaling for P/Invoke string arguments"`
+~
+
+### `description` property { #ruleDescriptor-description}
+
+An `ruleDescriptor` object should contain a property named `description` whose value is a string that describes the rule.
+
+The `description` property should, as far as possible, provide details sufficient to enable resolution of issues. 
+
+The first sentence of the `description` property should provide a concise description of the rule,
+suitable for display in cases where UI is limited.
+Tools that construct `description` in this way need not provide a value for the `abstract` property.
+Tools that do not construct `abstract` in this way should provide a value for the `abstract` property,
+because otherwise, the initial portion of `description` that a viewer displays where UI space is limited
+might not be understandable.
+
+### `options` property { #ruleDescriptor-options }
+
+A `ruleDescriptor` object may contain a property named `options`, whose value is a JSON object
+consisting of an arbitrary set of string/value pairs.
```

Comment:
(actually "string/value" was more common in the text so I settled on that instead.)